src/game/Grids/ObjectGridLoader.cpp

```diff
@@ -118,47 +118,61 @@ void LoadHelper(CellGuidSet const& guid_set, CellPair& cell, GridRefManager<T>&
 
     for (uint32 guid : guid_set)
     {
-        T* obj;
+        T* obj = nullptr;
```

Comment:
This is unnecessary. Should new fail, it will fail with exception. We populate the field with every flow.